mooncake-store/proto/master.proto

```diff
@@ -137,4 +147,7 @@ service MasterService {
 
   // Unmount a segment.
   rpc UnmountSegment(UnmountSegmentRequest) returns (UnmountSegmentResponse);
+
+  // Check existence of a key.
+  rpc ExistKey(ExistKeyRequest) returns (ExistKeyResponse);
```

Comment:
We can remove this file in the future, as it is no longer in use.

Reply:
Got it, Mooncake has discard the grpc, rigtht?

Reply:
Thats correct. We use coro_rpc as a replacement.